yt/visualization/base_plot_types.py

```diff
@@ -211,16 +211,20 @@ def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
         elif cbnorm == "linear":
             cbnorm_cls = matplotlib.colors.Normalize
         elif cbnorm == "symlog":
+            # if cblinthresh is not specified, try to come up with a reasonable default
+            vmin = float(np.nanmin(data))
+            vmax = float(np.nanmax(data))
             if cblinthresh is None:
-                cblinthresh = float((np.nanmax(data) - np.nanmin(data)) / 10.0)
-
-            cbnorm_kwargs.update(
-                dict(
-                    linthresh=cblinthresh,
-                    vmin=float(np.nanmin(data)),
-                    vmax=float(np.nanmax(data)),
-                )
-            )
+                # negative and positive range: choose val closest to zero
+                if vmin < 0 and vmax > 0:
+                    vmin_pos = np.nanmin(data[data > 0])
+                    vmax_neg = np.nanmax(data[data < 0])
+                    cblinthresh = np.min([vmin_pos, -vmax_neg])
+                # only positive or negative range: choose smallest non-zero number
+                else:
+                    cblinthresh = np.nanmin(np.absolute(data)[np.absolute(data) > 0])
```

Comment:
I am unsure this is necessary. Wouldn't the following simplification achieve exactly the same goal?
```suggestion
                cblinthresh = np.nanmin(np.absolute(data)[data != 0])
```

Reply:
agreed

Reply:
This is a great simplification!  Thank you!  I will add this when I figure out how to fix my darned PR.